src/UI/admin/components/hall-request/AdminHallRequestStyle.js

```diff
@@ -11,4 +11,504 @@ const HallReqTitle = styled.h2`
   color: blue;
 `;
 
-export { HallReqContainer, HallReqTitle };
+const HallDiv = styled.div`
+  margin: auto;
+  width: 1024px;
+  height: 352px;
+  border-radius: 4px;
+  border: solid 1px #d9d9d9;
+  background-color: #ffffff;
+  display: flex;
+  box-sizing: border-box;
+`;
+
+const ImageDiv = styled.div`
```

Comment:
ako je image  onda koristis styled.img to ti dozvoljava da passas src attribute u komponentu bez deklariranja propsova. (po defaultu styled.img prima src i alt attribute) tak da to promjeniti